web/main/models.py

```diff
@@ -3683,41 +3725,44 @@ def _delete_related_links_and_text_blocks(self):
             cls.objects.filter(id__in=ids).delete()
 
     @property
-    def sections(self):
-        return Section.objects.filter(casebook=self).filter(resource_type__isnull=True)
+    def sections(self) -> models.QuerySet[Section]:
+        return Section.objects.filter(casebook=self, resource_type__isnull=True)
```

Comment:
I think this method is wrong now (and has long been): there are now Sections that have `resource_type='Section'`, if I recall... but in any case, I don't think this syntax change makes it more wrong 😂